plugins/ocm-backend/src/providers/ManagedClusterProvider.ts

```diff
@@ -84,25 +95,29 @@ export class ManagedClusterProvider implements EntityProvider {
     );
 
     const resources: ResourceEntity[] = (
-      (await getManagedClusters(this.client)) as { items: Array<any> }
+      await listManagedClusters(this.client)
     ).items.map(i => {
+      const normalizedName =
+        i.metadata!.name === HUB_CLUSTER_NAME_IN_OCM
+          ? this.hubResourceName
+          : i.metadata!.name!;
+
       return {
         kind: 'Resource',
         apiVersion: 'backstage.io/v1beta1',
         metadata: {
-          name: i.metadata.name,
-          title: i.metadata?.labels?.name,
+          name: normalizedName,
+          title: i.metadata?.labels?.name || normalizedName,
```

Comment:
```suggestion
          title: normalizedName || i.metadata?.labels?.name,
```
This is probably the the reason why the hub cluster is named `local-cluster` in the catalog.